dashboard/15-final/app/ui/logo.tsx

```diff
@@ -0,0 +1,46 @@
+export function Logo() {
```

Comment:
I know @StephDietz put the logo in the public folder, we can do that or make it a component, no strong opinion.

Reply:
I think we landed on the same direction, let's make it a component :)